HW_3/hw03.Rmd

```diff
@@ -0,0 +1,170 @@
+# Homework 03
+
+You will analyze data looking at the relationship between **green reputation** and three personality traits--**compassion**, **intellectual curiosity**, and **openness to experiences**.
+The dataset includes data from **students** and **non-students**.
+
+```{r}
+dictionary <- readr::read_csv("green_dictionary.csv")
+green_data <- readr::read_csv("green_data.csv")
+library(dplyr) 
+library(ggplot2)
+library(tidyverse)
+``` 
+
+For your assignment, do the following.
+
+  1. Inspect the item responses (e.g., with graphs or by summarizing distinct values). 
+     Is anything unusual?
+```{r}
+summary(green_data) 
```

Comment:
Great job using the summary() function. I would recommend you to summarize the green_dictionary file as well!